test/types/holder.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+)
+
+// KogitoServiceHolder Helper structure holding informations which are not available in KogitoService
+type KogitoServiceHolder struct {
+	v1alpha1.KogitoService
+	Infinispan struct {
+		Username string
+		Password string
+	}
+}
+
+// KogitoBuildHolder Helper structure holding informations for Kogito build
+type KogitoBuildHolder struct {
+	*KogitoServiceHolder
+	*v1alpha1.KogitoBuild
```

Comment:
Why does the KogitoBuildHolder contains also KogitoServiceHolder? Because the KogitoService part is used for Kogito Runtime configuration?

Reply:
When user use CLI command `kogito deploy` (without specifying image to be used) then Kogito CLI creates both KogitoBuild and KogitoRuntime CRs. 
To replicate this behaviour in test I also create both KogitoBuild and KogitoRuntime in one step.
KogitoServiceHolder in this case holds KogitoRuntime struct.

Reply:
I see, so `kogito deploy` command takes parameters from which some are used for KogitoBuild (currently only native) and some of them for KogitoRuntime (runtime parameters like Infinispan configuration etc.), right?

Reply:
Yes